extension.neon

```diff
@@ -11,3 +11,11 @@ services:
 		class: bitExpert\PHPStan\Magento\Reflection\Framework\DataObjectMagicMethodReflectionExtension
 		tags:
 			- phpstan.broker.methodsClassReflectionExtension
+	-
+		class: bitExpert\PHPStan\Magento\Rules\AbstractModelUseServiceContractRule
+		tags:
+			- phpstan.rules.rule
+	-
+		class: bitExpert\PHPStan\Magento\Rules\AbstractModelRetrieveCollectionViaFactoryRule
+		tags:
+			- phpstan.rules.rule
```

Comment:
As long as you don't need any non-autowirable arguments, you can put these classes into top-level `rules` section like here: https://github.com/phpstan/phpstan-src/blob/f578cd1db76f21ed5d0397e9de32bc881ca9cfbf/conf/config.level0.neon#L14-L36

Reply:
Thanks I had seen extensions do it both ways and was not sure why. Makes sense.